inc/thorvg.h

```diff
@@ -1774,11 +1774,12 @@ class TVG_API WgCanvas final : public Canvas
     /**
      * @brief Sets the drawing target for the rasterization.
      *
+     * @param[in] device WGPUDevice, a desired handle for the wgpu device. If it is @c nullptr, ThorVG will assign an appropriate device internally.
      * @param[in] instance WGPUInstance, context for all other wgpu objects.
-     * @param[in] surface WGPUSurface, handle to a presentable surface.
+     * @param[in] surface or texture either WGPUSurface or WGPUTexture, serving as handles to a presentable surface or texture.
```

Comment:
`target Either WGPUSurface or WGPUTexture, serving as handles to a presentable surface or texture.`